app/controllers/idv/opt_in_ipp_controller.rb

```diff
@@ -0,0 +1,93 @@
+module Idv
+  class OptInIppController < ApplicationController
+    # include AcuantConcern
+    # include DocumentCaptureConcern
+    include IdvStepConcern
+    include StepIndicatorConcern
+    include PhoneQuestionAbTestConcern
+
+    # todo: do we need rate limiting here?
+    # before_action :confirm_not_rate_limited
+    # before_action :confirm_in_person_needed
+    # before_action :override_csp_to_allow_acuant
+    # todo: add a check_or_render_not_found guard
+
+    def show
+      analytics.idv_doc_auth_document_capture_visited(**analytics_arguments)
+
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('document_capture', :view, true)
+
+      render :show, locals: extra_view_variables
+    end
+
+    def update
+      idv_session.redo_document_capture = nil # done with this redo
+      # Not used in standard flow, here for data consistency with hybrid flow.
+      document_capture_session.confirm_ocr
+
+      result = handle_stored_result
+      analytics.idv_doc_auth_document_capture_submitted(**result.to_h.merge(analytics_arguments))
+
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('document_capture', :update, true)
+
+      cancel_establishing_in_person_enrollments
+
+      if result.success?
+        redirect_to idv_ssn_url
+      else
+        redirect_to idv_document_capture_url
```

Comment:
todo: integrate with the new FlowPolicy helper. I think this basically involves:

1. Add a [step_info class method](https://github.com/18F/identity-idp/blob/9e63aad5a58712c1e62f32c5e9f2ec370c574716/app/controllers/idv/how_to_verify_controller.rb#L36-L45)
2. Add this page as [a new step in the FlowPolicy](https://github.com/18F/identity-idp/blob/bf6b8b3ec83afae2b4fa99a89aa759ddfe5ea534/app/policies/idv/flow_policy.rb#L35-L51)